docs/ExportFormats.md

```diff
@@ -0,0 +1,26 @@
+# Export Formats
+
+## Summary table CSV export
+
+The summary table CSV export includes the following columns:
+
+* `Phyloreference ID`: A URI (absolute, relative or local; may or may not resolve) that identifies this phyloreference. Normally intended for machine consumption.
+  * Example `#Alligatoridae`
+* `Label`: A label for this phyloreference, normally intended for human consumption.
+  * Example: `Alligatoridae`
+* `Type`: The type of this phyloreference.
+  * May be one of the following values:
+    * `Apomorphy-based clade definition`
+    * `Maximum clade definition`
+    * `Minimum clade definition`
+    * `Invalid definition (...)` (with the reason for the invalid definition in parentheses)
+* `Definition`: The (human readable) clade definition as entered in the `Definition in free-form text`.
+  * Example: `Last common ancestor of Crocodylus niloticus, Osteolaemus tetraspis, and Tomistoma schlegelii and all of its descendents.`
+* `Internal specifier N`: A description of each internal specifier (_N_ is a number starting from 1).
+  * Example: `Caiman crocodilus`
+* `External specifier N`: A description of each external specifier (_N_ is a number starting from 1).
+  * Example: `Caiman crocodilus`
+* `Phylogeny: [Phylogeny label] expected`: The label of the node in the specified phylogeny to which the phyloreference was expected to resolve by the author.
+  * Example: `Alligatoridae`
+* `Phylogeny: [Phylogeny label] actual`: The label of the node actually resolved in the specified phylogeny to which the phyloreference was expected to resolve by the author.
+  * Example: `Alligatoridae_ott195670`, or `(unlabelled)` if the node has no label
```

Comment:
Isn't it Tidy Data practice to have NA or an equivalent value representation to signify that there is no value? Why should we make up a special string to signify the absence of a value? Or is this the special value that's already present on the phylogeny?

Reply:
The idea behind `(unlabelled)` was to distinguish between the case where a phyloref doesn't resolve at all (I think we currently represent this as blank in the `Phylogeny: [Phylogeny label] actual` field) and the case where it does resolve, but it does so on a node that doesn't have a label. Some options:
- `NA` for unlabelled node and blank for no resolution
- `(unlabelled)` (or some other standard string) for unlabelled node and `NA` (or some other string, e.g. `(could not resolve)`, `None`) for no resolution
- `NA: unlabelled node` vs `NA: could not resolve`
- We could use the internal node identifier, which looks something like `#phylogeny0_node0`, if the resolved node is unlabelled and blank if the phyloref could not be resolved
- Add additional `Phylogeny: [Phylogeny label] error` column, which is blank unless the `... actual` column is `NA`.

What do you think?

Reply:
Perhaps this: _We could use the internal node identifier, which looks something like #phylogeny0_node0, if the resolved node is unlabelled and blank if the phyloref could not be resolved_?

But on reflection, using `(unlabelled)` and `(could not resolve)` seem at least as clear if not clearer.

Reply:
> But on reflection, using `(unlabelled)` and `(could not resolve)` seem at least as clear if not clearer.

Agreed! I've implemented this in 5464ea6b6012d92c90cfda15d328029feb8a6cef.

Reply:
I've made this a bit clearer in d9f349f:

* `Phylogeny: [Phylogeny label] expected`: The label of the node in the specified phylogeny to which the phyloreference was expected to resolve.
  * Example: `Alligatoridae`
  * Will be set to `(none)` if no expected node was found.
* `Phylogeny: [Phylogeny label] actual`: The label of the node actually resolved for this phyloreference in the specified phylogeny.
  * Example: `Alligatoridae_ott195670`
  * Will be set to `(resolution not yet run)` if resolution has not yet been run.
  * Will be set to `(could not resolve)` if this phyloreference could not be resolved.
  * Will be set to `(unlabelled)` if a node was resolved, but it had no label.